src/main/java/com/internet/shop/security/AuthenticationServiceImpl.java

```diff
@@ -13,12 +13,12 @@
 
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        final String error = "Incorrect login or password";
+        final String exceptionMessage = "Incorrect login or password";
         User user = userService.findByLogin(login).orElseThrow(() ->
-                new AuthenticationException(error));
+                new AuthenticationException(exceptionMessage));
         if (user.getPassword().equals(password)) {
```

Comment:
let's better check all conditions inside `if` so that we could throw our Exception only once